Sources/Models/Views/Label.swift

```diff
@@ -46,7 +46,7 @@ public struct Label: IBDecodable, ViewProtocol, IBIdentifiable {
     public let variations: [Variation]?
     public let backgroundColor: Color?
     public let tintColor: Color?
-    public let adjustsFontForContentSizeCategory: Bool?
```

Comment:
Do not remove things from other pr, you failed to merge I think
Do not force , take the time to take code from two branch when there is conflict